torch/csrc/jit/ir.h

```diff
@@ -171,6 +171,7 @@ using NodeKind = Symbol;
 struct Value {
   TH_DISALLOW_COPY_AND_ASSIGN(Value);
   Value(Node * node_, size_t offset_);
+  virtual ~Value() {}
```

Comment:
There is no need for any changes to ir.h to add default values. We already support default values in `FunctionSchema` and each `script::Method` object has a function schema.